<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlayerWidget</class>
 <widget class="QMainWindow" name="PlayerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ONVIF Player</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="0" column="2">
     <widget class="QWidget" name="video_canvas" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QGridLayout" name="video_layout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
      </layout>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QWidget" name="controls_canvas" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QGridLayout" name="controls_layout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
      </layout>
     </widget>
    </item>
    <item row="0" column="1" rowspan="2">
     <widget class="QWidget" name="frames_list_canvas" native="true">
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
      <layout class="QGridLayout" name="frames_list_layout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuVerify">
    <property name="title">
     <string>Verify</string>
    </property>
    <addaction name="actionFile_structure"/>
    <addaction name="actionFile_signature"/>
    <addaction name="separator"/>
    <addaction name="actionCertificate_storage"/>
   </widget>
   <widget class="QMenu" name="menuAdditional">
    <property name="title">
     <string>Additional</string>
    </property>
    <addaction name="actionMemory"/>
   </widget>
   <widget class="QMenu" name="menuRender">
    <property name="title">
     <string>Render</string>
    </property>
    <widget class="QMenu" name="menuVideo_streams">
     <property name="title">
      <string>Video streams</string>
     </property>
     <addaction name="action1"/>
    </widget>
    <widget class="QMenu" name="menuAudio_streams">
     <property name="title">
      <string>Audio streams</string>
     </property>
     <addaction name="action1_2"/>
    </widget>
    <addaction name="menuVideo_streams"/>
    <addaction name="menuAudio_streams"/>
    <addaction name="separator"/>
    <addaction name="actionLocalTime"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuRender"/>
   <addaction name="menuVerify"/>
   <addaction name="menuAdditional"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionFile_structure">
   <property name="text">
    <string>File structure...</string>
   </property>
  </action>
  <action name="actionFile_signature">
   <property name="text">
    <string>File signature...</string>
   </property>
  </action>
  <action name="actionMemory">
   <property name="text">
    <string>Memory</string>
   </property>
  </action>
  <action name="actionCertificate_storage">
   <property name="text">
    <string>Certificate storage</string>
   </property>
  </action>
  <action name="actionLocalTime">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show local time</string>
   </property>
  </action>
  <action name="actiontest">
   <property name="text">
    <string>test</string>
   </property>
  </action>
  <action name="action1">
   <property name="text">
    <string>1</string>
   </property>
  </action>
  <action name="action1_2">
   <property name="text">
    <string>1</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
